### ISSUE TITLE
Dropdown menu does not work correctly on keyboard tabulation

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#1141]
Dropdown items are not accessible using tabs

![dropdown](https://user-images.githubusercontent.com/7963804/44902927-899fd900-ad0b-11e8-8eba-a64108d04be3.gif)

We need to tab through all dropdown items.

Bootstrap is a good about we need:
https://getbootstrap.com/docs/4.1/components/dropdowns/#single-button

**Comment by carloslancha** [comment]
We need to create a focus manager to deal with this.

**Comment by bryceosterhaus** [comment]
I'll work on this

**Comment by matuzalemsteles** [comment]
We have implemented this in 2.x. Closing!
